src/main/java/serenitylabs/tutorials/vetclinic/playingball/model/GameSchedule.java

```diff
@@ -0,0 +1,12 @@
+package serenitylabs.tutorials.vetclinic.playingball.model;
+
+import java.time.LocalDate;
+
+/**
+ * Created by kimahale on 9/8/2016.
+ */
+public interface GameSchedule {
+    PlayBall forGameOn(LocalDate someDay);
+
+    PlayBall forGameOnByWeek(LocalDate currentDay);
```

Comment:
If you use Polymorphism, you should only need the forGameOn() method, not the forGameOnByWeek() one.